src/mhw_lookup_monster.h

```diff
@@ -9,155 +9,103 @@
 // https://github.com/Haato3o/HunterPie/blob/db774b871e39629dc1d4bd58754def4c556701a2/HunterPie/HunterPie.Resources/Data/MonsterData.xml
 
 namespace mhw_lookup {
+
+	enum crown_preset_names {Standard,Alternate,Savage,Rajang,Undefined};
+
+	struct crown_preset_data {
+		uint8_t		crown_preset_id;
+		float		mini;
+		float		silver;
+		float		gold;
+	};
+
+	crown_preset_data CROWN_PRESETS[] = {
```

Comment:
Can you make this `const`?

Reply:
I changed it to `const crown_preset_data CROWN_PRESETS[4];`.
Shouldn't `monster_data	MONSTERS[]` be `const` as well?

Reply:
No, because that `monster_data MONSTERS[]` gets sorted on the fly upon starting the program (in `sort_MONSTERS`).